src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-3DMARK.json

```diff
@@ -0,0 +1,36 @@
+{
+  "Description": "3DMark TimeSpy GPU Performance Workload",
+  "Actions": [
+    {
+      "Type": "ThreeDMarkExecutor",
+      "Parameters": {
+        "Scenario": "TimeSpy",
+        "LisenceKey": "",
+        "PackageName": "3dmark",
+        "Tags": "GPU,3DMark,Gaming"
+      }
+    }
+  ],
+  "Monitors": [
+    {
+      "Type": "PerfCounterMonitor",

```

Comment:
Yea, I wasn't sure which monitor to put here since users could potentially be running on Nvidia systems. Maybe make one profile for AMD and one for Nvidia?